src/staticdata.c

```diff
@@ -3367,11 +3376,14 @@ JL_DLLEXPORT void jl_create_system_image(void **_native_data, jl_array_t *workli
     int64_t datastartpos = 0;
     JL_GC_PUSH4(&mod_array, &extext_methods, &new_ext_cis, &edges);
 
+    jl_query_cache query_cache = { 0 };
+    htable_new(&query_cache.type_in_worklist, 0);
```

Comment:
```suggestion
    jl_query_cache query_cache;
    htable_new(&query_cache.type_in_worklist, 0);
```

please add code to `htable_free` this also

Reply:
Ack, nice catch - I deleted this on accident with the logging code for the hit rate check